### PR TITLE
61119 Adding chip lib configuration settings

### PR DESCRIPTION
**Pull request by kanchanasuriya**
## Summary

- *Adding configuration settings for chip lib to integrate with mobile app*

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#61119

## Testing done
- [x] rspec
- [x] rubocop

## What areas of the site does it impact?
NA

## Acceptance criteria

- [X]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X]  No error nor warning in the console.
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by randomsync — APPROVED** [review]
lgtm

File: config/settings.yml [hunk 1]
```diff
@@ -1444,6 +1444,14 @@ github_cvu:
   integration_id: ~
   private_pem: ~
 
+chip:
+  url: https://vpce-06399548ef94bdb41-lk4qp2nd.execute-api.us-gov-west-1.vpce.amazonaws.com
+  base_path: dev
+  tmp_api_user: TzY6DFrnjPE8dwxUMbFf9HjbFqRim2MgXpMpMciXJFVohyURUJAc7W99rpFzhfh2B3sVnn4
+  tmp_api_id: 2dcdrrn5zc
+  tmp_api_username: vetsapiTempUser
+  mock: true
```
Comment (ryan-mcneil) [inline config/settings.yml:1453]: Just confirming none of these should be stored in Param store

Comment (kanchanasuriya) [reply config/settings.yml:1453]: Yes, it is not required for test environment

**Review by ryan-mcneil — DISMISSED** [review]
👌

**Review by ryan-mcneil — APPROVED** [review]
👌

**Pull request merged**